cmd/root/results.go

```diff
@@ -114,12 +201,11 @@ func numMistakes(events []event) (mistakes int) {
 	return
 }
 
-func duration(startTime time.Time, endTime time.Time) string {
-	nanos := (endTime.UnixMilli() - startTime.UnixMilli()) * int64(time.Millisecond)
-	d := time.Duration(nanos)
-
-	s := fmt.Sprintf("%.3fs", d.Seconds())
-	return s
+// Returns the duration between the first event and
+// the last event of the array. If there are less than
+// two events in the list, it returns zero duration.
+func duration(events []event) time.Duration {
```

Comment:
I haven't tested this, so I should write tests to verify the comment's statement